application/datavalueobjects/SimpleSurveyValues.php

```diff
@@ -25,6 +25,12 @@ class SimpleSurveyValues
     /** @var  int the surveygroup from which the new survey will inherit values */
     private $surveyGroupId;
 
+    /** @var string administrator name */
+    private $admin = 'inherit';
```

Comment:
There's no need to make this private if you have both a (trivial) getter and setter, I think.